src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,37 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's contract in the address book.
+ */
+
+public class Contact {
+	
+    public final String value;
+    private boolean isPrivate;
+    
+	public Contact(String value, boolean isPrivate) {
+		this.value = value;
```

Comment:
Notice that your indentation mechanism uses tabs for indentation. 
The coding standards we follow require using 4 spaces for indentation.
Tip: you can set Eclipse to do this for you, check out how!